layouts/shortcodes/asm-libraries-capabilities.md

```diff
@@ -4,9 +4,9 @@ The following ASM capabilities are supported relative to each language's tracing
 | ASM capability                         | Java    | .NET     | Node.js                                                                     | Python        | Go              | Ruby          | PHP           |
 |----------------------------------------|---------|----------|-----------------------------------------------------------------------------|---------------|-----------------|---------------|---------------|
 | Threat Detection                       | 1.8.0   | 2.23.0   | 3.13.1                                                                      | 1.9.0         | 1.47.0          | 1.9.0         | 0.84.0        |
-| API Security                           | 1.31.0  | 2.42.0   | 3.51.0 for Node.js 14+, or 4.30.0 for Node.js 16+, or 5.6.0 for Node.js 18+ | not supported | 1.59.0          | 1.15.0        | 0.98.0        |
+| API Security                           | 1.31.0  | 2.42.0   | 3.51.0 for Node.js 14+, 4.30.0 for Node.js 16+, or 5.6.0 for Node.js 18+ | not supported | 1.59.0          | 1.15.0        | 0.98.0        |
 | Threat Protection                      | 1.9.0   | 2.26.0   | 3.19.0                                                                      | 1.10.0        | v1.50.0         | 1.11.0        | 0.86.0        |
-| Customize response to blocked requests | 1.11.0  | 2.27.0   | 3.22.0 for Node.js 14+, 4.1.0 for Node.js 16+                               | 1.19.0        | v1.53.0         | 1.15.0        | 0.86.0        |
+| Customize response to blocked requests | 1.11.0  | 2.27.0   | 3.22.0 for Node.js 14+, or 4.1.0 for Node.js 16+                               | 1.19.0        | v1.53.0         | 1.15.0        | 0.86.0        |
```

Comment:
not sure why you removed the `or` from API Security line to add it back in this line ? looks like bad copy paste ?

Reply:
oh because it's the last one in the line ok i get it